packages/atlas/src/views/studio/VideoWorkspace/NFTForm/types.ts

```diff
@@ -0,0 +1,10 @@
+import { NftAuctionInputMetadata, NftIssuanceInputMetadata } from '@/joystream-lib'
+
+export type Listing = 'Auction' | 'Fixed price' | 'Not for sale' | undefined
+
+export type AuctionDate = {
+  startDate: Date | string | null
+  endDate: Date | string | null
+}
+
+export type NFTFormData = NftAuctionInputMetadata & NftIssuanceInputMetadata & AuctionDate
```

Comment:
You include both `NftAuctionInputMetadata` and `NftIssuanceInputMetadata` here, this will create quite a complex data structure with some unneeded fields. Also the nullability of some the fields will not match. I think we should just have independent type here, something like this should be enough?
```suggestion
export type NFTFormData = {
  royalty?: number
  startingPrice?: number
  buyNowPrice?: number
  whitelistedMembersIds?: string[] // to be used later
} & AuctionDate
```